texajson.go

```diff
@@ -189,7 +253,7 @@ func dupCount(list []string) map[string]int {
 }
 
 // ConvtoSlabPage configures the parameters of SlabPage using the ArtiQSA
-func ConvtoSlabPage(ArtiQSA []uint64, SlabNameArray []string, slabSeqArray []string) []SlabPage {
+func ConvtoSlabPage(ArtiQSA []uint64, SlabNameArray []string, slabSeqArray []string, justification []string, transactions []string) []SlabPage {
```

Comment:
Since there are more than one justification in a given session, it makes more sense to address the plurality as `justifications`?

Reply:
Done.